e2e/docker-containers-health-check.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
Is there any reason to use this script instead of writing it in TypeScript? With TS it would be much more maintainable.

Reply:
It's not the concern of the TS code; also easier to test this code from command line when it's a shell script.

Reply:
Replaced shell script code with ts code for docker healthcheck.